### ISSUE TITLE
Cannot figure out why split a batch into two halves

### ISSUE THREAD

**Issue by pyun-ram** [kshmelkov/incremental_detectors#9]
Let's see the two lines in https://github.com/kshmelkov/incremental_detectors/blob/master/network.py#L36-L37

Why does it split the batch into two halves one for logits and another for logits_for_distiallation?

**Comment by pyun-ram** [comment]
Sorry It was my bad. I figure it out you merge the rois and proposals in https://github.com/kshmelkov/incremental_detectors/blob/master/network.py#L127
